src/tasks/start/backend/create-app.ts

```diff
@@ -7,37 +7,27 @@ import {
   RepoContract,
   RepoInstance,
   APMRegistryContract,
-  APMRegistryInstance
+  APMRegistryInstance,
+  AppStubInstance
 } from '~/typechain'
 
-interface InitializableApp extends Truffle.ContractInstance {
-  initialize: (...args: any[]) => void
-}
-
 export async function createApp(
   appName: string,
   appId: string,
   dao: KernelInstance,
-  proxyInitParams: any[],
```

Comment:
Proxies are no longer initialized in app creation. This is useful for apps like the Token manager, which need to operate with the proxy (setting it as the controller of a token) before actually initializing it.